### PR TITLE
Added initial otterdog configuration for the opendut main repository.

### PR DESCRIPTION
**Pull request by kKdH**

### CODE REVIEW

**Comment by netomi** [comment]
You can also use the playground to sketch changes prior to making a PR: https://eclipse-opendut.github.io/.eclipsefdn/playground/

Some settings dont have to be specified as they are already inherited from the default config, when using the playground you see exactly what the default values are.

File: otterdog/eclipse-opendut.jsonnet [hunk 1]
```diff
@@ -19,6 +19,30 @@ orgs.newOrg('eclipse-opendut') {
     },
   },
   _repositories+:: [
+    orgs.newRepo('opendut') {
+      description: "Test Electronic Control Units around the world in a transparent network.",
+      private: true,
```
Comment (netomi) [inline otterdog/eclipse-opendut.jsonnet]: are you sure you want to create a private repo?

Comment (kKdH) [reply otterdog/eclipse-opendut.jsonnet]: Yes. I'm going to switch to public when we are ready.

Comment (netomi) [reply otterdog/eclipse-opendut.jsonnet]: In general thats not such a good idea. Private repos have some limitations that you need to be aware of while we are still on a free plan. Thats also quite unusual for doing open-source.

I would suggest that you create the repo as public, then you can already fork it, work on it in your fork, and once you think its worthwhile to merge it back to the upstream repo, create a PR with your changes.

**Comment by netomi** [comment]
the project just started so things are certainly a bit more relaxed, but could we get some project lead here on board such that everybody knows whats going on? An approval or +1 to this PR is good enough for me.

**Comment by jbaumgartl** [comment]
Review done

**Review by jbaumgartl — APPROVED** [review]

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
repo has been created. It might take 1-2 h for the permissions to be setup correctly, but you can already fork the repo.

**Pull request merged**